.env

```diff
@@ -24,3 +24,6 @@ MONGODB_URI=
 # For using Pinecone (store)
 PINECONE_API_KEY=
 PINECONE_HOST=
+
+# Some examples are expensive to run, so we disable them by default
+RUN_EXPENSIVE_EXAMPLES=false
```

Comment:
:D love it

Reply:
maybe rather an option on the example runner?

Reply:
I think its safe here, at least I, run the example one by one via PHP instead of the runner

Reply:
> :D love it

I think those examples are key and the best documentation for the users